core/templates/dev/head/pages/exploration-editor-page/preview-tab/preview-tab.directive.ts

```diff
@@ -165,6 +165,9 @@ angular.module('oppia').directive('previewTab', [
           };
 
           ctrl.resetPreview = function() {
+            if ($window.sequenceOfInteractions) {
```

Comment:
Please don't define new global variables on $window. Create a new service for this instead.

Reply:
Sure. The reason I used $window is because we would get rid of this once we obtain more detailed logs.

Reply:
Yep, but we still have a policy of not polluting the global namespace, so we should stick to that. (We don't know how long it will take to solve the issue, too.)

Reply:
Done. PTAL @seanlip.